packages/drivers/odsp-driver/src/test/socketTests/socketMock.ts

```diff
@@ -42,6 +42,9 @@ export interface IMockSocketConnectResponse {
 		  }
 		| {
 				eventToEmit: "connect_timeout";
+		  }
+		| {
+				eventToEmit: undefined;
```

Comment:
What's the rationale for an undefined event here? Isn't there a "real" event we should be mocking?

Reply:
Undefined if for the case, where we want to mock not sending any anything from service to client as handled in the switch statement.